### ISSUE TITLE
A11y_AzureDataStudioAAE_New Instance_Select the deployment options_ScreenReader: Screenreader announces unnecessary information while navigating with help of tab key. 

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15751]
### Actual:
While navigating with help of tab key and focus is on 'All' list. NVDA announces 'All one of five Level one', NVDA announces unnecessary information.
Similar issue is observed with JAWS.

### Expected:
While navigating with help of tab key and focus is on 'All' list. NVDA should announce 'All one of five' only.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#STP;#WCAG1.3.1;#NVDA;#Jaws;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab shift+f10 and press enter on "manage" menu item
5. Start NVDA
6. Tab to "new instance" button and press enter
7. Check how NVDA is announcing the information 

### User Impact:
Screenreader user will get confuse about information which is announced.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

https://user-images.githubusercontent.com/85863999/127466726-6213d807-4d57-438c-af58-c02bc8079232.mp4

**Comment by krrish2021** [comment]
Closing as not repro
![Screenshot (188)](https://user-images.githubusercontent.com/85863543/150970852-fdd28f84-d037-4a31-9210-6e96d7b4ff21.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
